cpplint/cpplint.py

```diff
@@ -2653,7 +2653,7 @@ def Update(self, filename, clean_lines, linenum, error):
     #   };
     class_decl_match = Match(
         r'^(\s*(?:template\s*<[\w\s<>,:]*>\s*)?'
-        r'(class|struct)\s+(?:[A-Z_]+\s+)*(\w+(?:::\w+)*))'
+        r'(class|struct)\s+(?:[A-Z0-9_]+\s+)*(\w+(?:::\w+)*))'
```

Comment:
C++ names cannot start with a digit, so why do this?

Reply:
This is about class decorators, such as "class V8_EXPORT_PRIVATE Foo {". The "V8_EXPORT_PRIVATE" is not matched currently.